src/resources/lib/pdfjs/web/viewer.html

```diff
@@ -42,6 +42,9 @@
   </head>
 
   <body tabindex="1">
+    <div hidden>
+      <canvas id="COVER_RENDER_CANVAS"></canvas>
+    </div>
```

Comment:
Hello Pierre @panaC 
When your PR is ready, we will make sure to update the actual source code of Thorium's PDF.js fork:
https://github.com/edrlab/pdf.js/blob/c745bd4060d217f712ecf4a2b57561b09fd9f905/web/viewer.html#L74-L75
Then we will update the README to reference the correct Git commit hash(es): https://github.com/edrlab/thorium-reader/blob/develop/src/resources/lib/pdfjs/README.md
(no need to update the TypeScript type definitions, as the JS source code has not changed: https://github.com/edrlab/thorium-reader/blob/develop/src/typings/pdf.js/README.md )

Reply:
Obsolete comment